apps/assessment_registry/schema.py

```diff
@@ -262,13 +263,20 @@ class Meta:
 
 
 class AdditionalDocumentType(DjangoObjectType, UserResourceMixin, ClientIdMixin):
+    external_link = graphene.String(required=False)
+
     class Meta:
         model = AdditionalDocument
-        fields = ("id", "file", "external_link")
+        fields = ("id", "file")
 
     document_type = graphene.Field(AssessmentRegistryDocumentTypeEnum, required=True)
     document_type_display = EnumDescription(source='get_document_type_display', required=True)
 
+    def resolve_external_link(root, info, **kwargs):
+        if root.external_link == "":
+            return None
+        return root.external_link
```

Comment:
Let's define a utils function. Something like this

utils/common.py
```python3
def render_string_for_graphql(text):
    """
    Return null if text is empty ("")
    """
    if text == '':
        return None
    return text
```

```suggestion
        if render_string_for_graphql(root.external_link)
```